scale/scheduler/tasks/services/messaging/messaging_service.py

```diff
@@ -0,0 +1,34 @@
+"""Defines the messaging service that handles the backend messaging"""
+from __future__ import unicode_literals
+
+import logging
+
+from scheduler.manager import scheduler_mgr
+from scheduler.tasks.services.messaging.message_handler_task import MessageHandlerTask
+from scheduler.tasks.services.service import Service
+
+
+logger = logging.getLogger(__name__)
+
+
+class MessagingService(Service):
```

Comment:
Didn't see any unit test coverage for this class.

Reply:
Unit tests for this class were added